functions/github-slack-announce-pull-requests-in-channel/Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+ruby '2.1.5'
```

Comment:
traveling ruby in ruby-on-lambda is currently 2.1.5, this could be updated, isn't terribly important.